tests/test_e2e_40_sdntrace.py

```diff
@@ -283,7 +283,7 @@ def test_020_run_sdntrace_fail_missing_flow(self):
         ]
 
         assert full_path != actual, f"Full path {full_path}. Actual: {actual}"
-        assert full_path[:4] == actual, f"Expected {full_path[:4]}. Actual: {actual}"
+        assert full_path[:3] == actual, f"Expected {full_path[:4]}. Actual: {actual}"
```

Comment:
The message for this assertion should have the slice updated for `full_path` to `:3`

Reply:
good catch, @Ktmi. I've added commit f49d7af addressing this problem. Please let me know if you have further comments.